blog/models.py

```
@@ -5,10 +5,11 @@
 from django.db import models
 from django.db.models import CASCADE, SET_NULL
 
+POST_IMAGE_UPLOAD_PATH = "images/posts/"
 
 def post_image_upload_to(instance, filename):  # s3의 images/posts/에 업로드 하는 함수
```

Ruff (0.15.5)

[warning] Unused function argument: `instance`

(ARG001)